src/components/forms/PlanForm/components/tests/retirePlanForm.test.tsx

```diff
@@ -0,0 +1,136 @@
+import { mount, shallow } from 'enzyme';
+import MockDate from 'mockdate';
+import React from 'react';
+import { act } from 'react-dom/test-utils';
+import { PlanDefinition } from '../../../../../configs/settings';
+import * as errors from '../../../../../helpers/errors';
+import { expectedPlanDefinition } from '../../tests/fixtures';
+import { RetirePlanForm } from '../retirePlanForm';
+import { retirePayload } from './fixtures';
+
+/* tslint:disable-next-line no-var-requires */
+const fetch = require('jest-fetch-mock');
+MockDate.set('1/30/2000');
+jest.mock('../../../../../configs/env');
+
+const props = {
+  cancelCallBack: jest.fn(),
+  payload: expectedPlanDefinition as PlanDefinition,
+  savePlan: jest.fn(),
+  setSubmittingCallBack: jest.fn(),
+};
+
+describe('forms/PlanForm/components/retirePlanForm', () => {
```

Comment:
Do we need a test that the plan endpoint is called with the right payload?

Reply:
Addressed [here](https://github.com/onaio/reveal-frontend/pull/1493/commits/3031856130992d8265637e99adef98da3bb4313b)